### DISCUSSION TITLE
Automatic restart of locast2plex Docker container

### DISCUSSION THREAD

**Discussion by SJLBoulder** [tgorgdotcom/locast2plex#166] [Show and tell]
Thanks, tgorgdotcom, for locast2plex.  I found it this weekend, and so far, it's working great for me.  If this proves reliable, I'll be dropping my basic cable TV subscription.

I wanted locast2plex to restart automatically when I restart my PLEX server (Ubuntu 18.04).  I found that this could be accomplished by adding an option "--restart unless-stopped" to the Docker run command:

`docker run --restart unless-stopped -v <full path to config file>:/app/config/config.ini -p 6077:6077 -p 1900:1900/udp --name locast2plex tgorg/locast2plex`

Also, I added the "--name" option so that the Docker image would be usefully named (rather than random).

**Comment by tgorgdotcom** [comment]
Nice catch.  I was using the restart option in my personal installation, but I forgot to mention it in the README.  Just added.
